schulcloud/FileHelper.swift

```diff
@@ -18,12 +18,16 @@ class FileHelper {
         return URL(string: "users/\(userId)/")!
     }
     
-    static var rootFolder: File = {
-        let fetchRequest: NSFetchRequest<File> = File.fetchRequest()
+    static var rootFolder: File {
+        return createRootFolder()
+    }
+    
+    static func createRootFolder() -> File {
+        let fetchRequest = NSFetchRequest(entityName: "File") as NSFetchRequest<File>
```

Comment:
Unfortunately, `.fetchRequest()` is buggy. The underlying entity descriptions are bound to the old context, which is discarded when the database is reset. I checked, `File.entity().name` is `nil` after the context has been discarded, so we're back to hardcoding strings.